hive3/src/main/java/org/apache/iceberg/mr/hive/vector/CompatibilityHiveVectorUtils.java

```diff
@@ -67,10 +66,11 @@ public static MapWork findMapWork(JobConf job) {
       LOG.debug("Initializing for input {}", inputName);
     }
     String prefixes = job.get(DagUtils.TEZ_MERGE_WORK_FILE_PREFIXES);
-    if (prefixes != null && !StringUtils.isBlank(prefixes)) {
-      // Currently SMB is broken, so we cannot check if it's  compatible with IO elevator.
-      // So, we don't use the below code that would get the correct MapWork. See HIVE-16985.
-      return null;
+    if (prefixes != null && !prefixes.trim().isEmpty()) {
+        // Currently SMB is broken, so we cannot check if it's  compatible with IO elevator.
+        // So, we don't use the below code that would get the correct MapWork. See HIVE-16985.
+        return null;
+      }
```

Comment:
I think } is an extra one and needs to be removed?

Reply:
yep you are right, seems to be weird because yesterday I compiled everything.